test_remove_due_date_alt.py

```diff
@@ -0,0 +1,84 @@
+from pytest import fixture
+from TodoList import TodoList
+import sys
+
+
+# Arrange
+
+@fixture
+def myList():
+    #Arrange
+    myList = TodoList()
+    myList.tasks = [("Lunch", "Small", "2025-4-23", [], "3.14159", False)]
+    return myList
+
+# Arrange
+@fixture
+def get_remove_date():
+    return 1
+
+# Arrange
+@fixture
+def get_invalid_task_num():
+    return 99
+
+# Arrange
+@fixture
+def get_no_due_date_task():
+    return 1
+
+def test_remove_due_date(get_remove_date, capfd):
+    out, err = capfd.readouterr()
+    # Save the original stdin
+    original_stdin = sys.stdin
+    
+    # Create a new StringIO object with the inputs
+    sys.stdin = StringIO(str(get_remove_date))
```

Comment:
All instances of `StringIO` throw this error:

```
>       sys.stdin = StringIO(str(get_no_due_date_task))
E       NameError: name 'StringIO' is not defined

test_remove_due_date_alt.py:73: NameError
```

## Proposed change

Adding this import solves the above error:

```
from io import StringIO
```